lib/dnsimple/struct/tld.rb

```diff
@@ -30,6 +30,9 @@ class Tld < Base
 
       # @return [Boolean] True if DNSimple supports inbound transfers for this TLD.
       attr_accessor :transfer_enabled
+
+      # @return [String | nil] Type of data interface required for DNSSEC for this TLD.
```

Comment:
`|` is not a correct syntax. You can see warning running `yard server --reload`.

```suggestion
      # @return [String, nil] Type of data interface required for DNSSEC for this TLD.
```

Reply:
👍